grep-00XX-relicense-volk.md

```diff
@@ -53,6 +53,100 @@ We consider it to be very important to let every contributor know that the wish
 A first step would be to notify every contributor of this GREP.
 
 
+#### Email draft
+
+
+Dear contributors of VOLK,
+
+we are writing to let you know that we are working towards a new major release
+of VOLK. The possibly biggest change will be that we intend to use a different
+license for this release: VOLK 3.0 shall be licensed under the LGPL 3.0 (or
+later). To do this effectively, we will need your help, should you agree. If you
+are already aware of this effort and are in support, you can skip to "What are
+you asking from contributors?" below, but we recommend you read this letter in
+its entirety to make sure you understand our intentions and process.
+
+##### Why are we doing this?
+
+In retrospect, LGPL would probably have been the better choice for VOLK from the
+get-go. The FSF recommendations on libraries states that "If developers are
+already using an established alternative library released under a nonfree
+license or a lax pushover license, then we recommend using the GNU Lesser
+General Public License (LGPL)."
+(https://www.gnu.org/licenses/license-recommendations.html).
+
+We agree with this recommendation, and want to avoid VOLK being a SIMD library
+exclusively used by GNU Radio (and a few others), while other software packages
+rally behind other SIMD libraries, that are more liberally licensed.
+
+We could have chosen an even less restrictive license, but we are believers in
+copyleft, and believe the LGPL strikes the right balance between not restricting
+users in which types of software they may use VOLK, and requiring users to
```

Comment:
nah, can't say we require upstreaming – we encourage. LGPL can't force you to upstream anything.